usage/importing-history.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Importing your History
+parent: Usage
+permalink: /usage/importing-history/
+---
+
+FreeTube has the ability to import your history from a variety of services. Here you will find instructions on how to import from each service.
+
+## From YouTube
+
+1. Login to your YouTube account
+2. Click on your profile picture in the top right corner of the web page
+3. Click on "Your data in YouTube" in the displayed drop down
+4. Click on "More" in the "Your YouTube dashboard" card
+5. Click on "Download YouTube data"
+6. Click on "All YouTube data included" and deselect everything except for "History"
+7. Click "Next step"
+8. Change the History file format to "JSON"
+9. Click on "OK"
+10. Click "Next step"
+11. Select your preferred method of deliver (Email, Dropbox, etc.) and click on "Create Export" (Instructions from here on out will assume that you selected email)
+12. Check your email and download the .zip file that is provided.
+13. Extract the .zip file
+14. The file you want will be in `Takeout/YouTube and YouTube Music/history/history.json`
```

Comment:
cant verify that this is the right path

Reply:
1. The path name varies on your locale
2. not `history.json` but `watch-history.json`, there is also a `search-history.json` we don't use

Reply:
so full path is ``Takeout/YouTube and YouTube Music/history/watch-history.json``

Reply:
I got Chinese one ._.
![image](https://user-images.githubusercontent.com/1018543/225845730-38e1d155-c59e-4ddd-b4db-4fb16ba39dc4.png)

Let me see if I can get an English one

Reply:
Confirmed
![image](https://user-images.githubusercontent.com/1018543/225848279-9a4647ff-63ef-4f6d-881b-d37d200c3acb.png)